components/authentication/base/admin-checker/external-secrets/rhtap-infra-secrets.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: rhtap-infra-secrets
+  annotations:
+    argocd.argoproj.io/sync-options: SkipDryRunOnMissingResource=true
+    argocd.argoproj.io/sync-wave: "-1"
```

Comment:
You might be able to use a later sync-wave here, like the default of 0.

Reply:
I did not check them all but all the ExternalSecret I did check use sync-wave -1 so I used -1 as well. I guess that if we use same sync wave as the one creating the conjob, it might complains that secret is not defined. Let me test that.